src/appleseed.studio/mainwindow/rendering/materialdrophandler.cpp

```diff
@@ -78,6 +78,12 @@ namespace
     {
         std::string                     m_name;
         renderer::ObjectInstance::Side  m_side;
+
+        explicit MaterialSlot(const std::string name, const renderer::ObjectInstance::Side side)
```

Comment:
I'm surprised that this compiles without a default constructor, but if it does, then fine :)

Reply:
Hi @dictoon , 
Yes it does. Do you want me to rebase this PR?

Reply:
I checked recent commits and it looks like we are going to drop 2012. Do you want me to delete this PR instead?

Reply:
Yes, it's not longer needed. Thanks for your work though!